capstone/src/test/java/com/google/sps/servlets/clubs/EditClubServletTest.java

```diff
@@ -0,0 +1,126 @@
+package com.google.sps.servlets;
+
+import static org.mockito.Mockito.when;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.FilterOperator;
+import com.google.appengine.api.datastore.Query.FilterPredicate;
+import com.google.appengine.tools.development.testing.LocalBlobstoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import com.google.appengine.tools.development.testing.LocalUserServiceTestConfig;
+import com.google.common.collect.ImmutableList;
+import java.io.IOException;
+import java.util.ArrayList;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+@RunWith(JUnit4.class)
+public class EditClubServletTest {
+
+  private final String SAMPLE_CLUB_NAME = "Test Club";
+  private final String SAMPLE_CLUB_DESC_1 = "Test club description";
+  private final String SAMPLE_CLUB_WEB = "www.test-club.com";
+  private final String TEST_EMAIL = "test-email@gmail.com";
+
+  @Mock private HttpServletRequest request;
+  @Mock private HttpServletResponse response;
+  private DatastoreService datastore;
+  private EditClubServlet editClubServlet;
+
+  private LocalServiceTestHelper helper =
+      new LocalServiceTestHelper(
+          new LocalDatastoreServiceTestConfig().setDefaultHighRepJobPolicyUnappliedJobPercentage(0),
+          new LocalUserServiceTestConfig(),
+          new LocalBlobstoreServiceTestConfig());
+
+  @Before
+  public void setUp() throws IOException {
+    helper.setUp();
+    MockitoAnnotations.initMocks(this);
+    this.editClubServlet = new EditClubServlet();
+    datastore = DatastoreServiceFactory.getDatastoreService();
+  }
+
+  @After
+  public void tearDown() {
+    helper.tearDown();
+  }
+
+  @Test
+  public void doPost_editClubAllValidOfficers() throws ServletException, IOException {
+    prepClubEnv();
+    String description = "new test description";
+    String website = "new-website.com";
```

Comment:
super nit: can you change these variable names to newDescription and newWebsite? It'll make it easier to understand that these are the new edits